libraries/RC_Channel/RC_Channels_VarInfo.h

```diff
@@ -92,5 +92,12 @@ const AP_Param::GroupInfo RC_Channels::var_info[] = {
     // @Bitmask: 0:Ignore RC Receiver, 1:Ignore MAVLink Overrides, 2:Ignore Receiver Failsafe, 3:FPort Pad, 4:Log RC input bytes, 5:Arming check throttle for 0 input, 6:Skip the arming check for neutral Roll/Pitch/Yay sticks, 7:Allow Switch reverse
     AP_GROUPINFO("_OPTIONS", 33, RC_CHANNELS_SUBCLASS, _options, (uint32_t)RC_Channels::Option::ARMING_CHECK_THROTTLE),
 
+    // @Param: _PROTOCOLS
+    // @DisplayName: Enabled RC protocols
```

Comment:
I would change the param Display name to just "RC Protocols enabled" to keep "RC" at the front.

For the Description I think in general we should avoid having "This controls" or "This is" because otherwise all parameters would have this.  I think we should also avoid, "You" by using the 3rd person tense (?).  So how about this alternative:

@Description: Bitmask of enabled RC protocols.  Allows narrowing the protocol detection to only specific types of RC receivers which can avoid issues with incorrect detection.  Set to 1 to enable all protocols.

Reply:
I've changed both, thanks